commercial/app/views/hosted/guardianHostedPage.scala.html

```diff
@@ -29,10 +29,10 @@
                       data-block-video-ads="true"
                       poster="@page.video.posterUrl"
                       class="vjs-hosted__video hosted__video gu-media--video vjs vjs-tech-html5 vjs-paused vjs-controls-enabled vjs_video_1-dimensions vjs-user-active">
-                        <source type="video/mp4" src="@page.video.srcUrl">
-                        <source type="video/webm" src="@page.video.srcUrl">
-                        <source type="video/ogg" src="@page.video.srcUrl">
-                        <source type="video/m3u8" src="@page.video.srcUrl">
+                        <source type="video/mp4" src="@page.video.srcUrl&format=video/mp4&maxbitrate=2048">
+                        <source type="video/webm" src="@page.video.srcUrl&format=video/webm&maxbitrate=2048">
+                        <source type="video/ogg" src="@page.video.srcUrl&format=video/ogg&maxbitrate=2048">
+                        <source type="video/m3u8" src="@page.video.srcUrl&format=video/m3u8&maxbitrate=2048">
```

Comment:
We could do this in the scala - potentially creating something like a:
`@page.video.srcUrlFor("mp4", 2048)`... not a biggy though.